lib/smart_answer_flows/uk-benefits-abroad.rb

```diff
@@ -612,13 +612,15 @@ def define
               outcome :jsa_eea_going_abroad_maybe_outcome
             elsif calculator.benefit == "winter_fuel_payment"
               outcome :wfp_going_abroad_eea_maybe_outcome
+            elsif calculator.benefit == "esa"
+              outcome :esa_going_abroad_eea_outcome
             end
           when "after_jan_2021"
-            if calculator.benefit == "jsa" || calculator.benefit == "winter_fuel_payment"
+            if calculator.benefit == "jsa" || calculator.benefit == "winter_fuel_payment" || calculator.benefit == "esa"
```

Comment:
At this point it becomes easier to change to:

```suggestion
            if %w[esa jsa winter_fuel_payment].include?(calculator.benefit)
```